### PR TITLE
bpo-33440: Defer imports in pathlib to reduce its import time.

### PR DESCRIPTION
**Pull request by agfor**
https://bugs.python.org/issue33440

### CODE REVIEW

File: Lib/pathlib.py
Comment (serhiy-storchaka) [inline Lib/pathlib.py]: What is the reason of the renaming?

Comment (matrixise) [reply Lib/pathlib.py]: there is no reason, you are right but this is just a copy. Now he could remove the renaming.

**Review by matrixise — APPROVED** [review]

**Comment by matrixise** [comment]
@serhiy-storchaka I can't merge this PR, if you want to merge it, let's go! ;-)

**Comment by serhiy-storchaka** [comment]
I don't know whether this change is worth merging.

**Comment by matrixise** [comment]
ok thanks for your feedback @serhiy-storchaka

**Review by pitrou — CHANGES_REQUESTED** [review]
Sorry, it looks like this review took a long delay for a simple change. This looks good to me on the principle but you need to do two things:
* merge / rebase your changes on git master
* add a NEWS entry for the change (you can use [blurb-it](https://blurb-it.herokuapp.com/) or [blurb](https://pypi.org/project/blurb/))

Also, would be nice if you posted import benchmark numbers for the change.

**Comment by csabella** [comment]
@agfor, please take a look at the review comments from pitrou.  Thanks!

**Comment by AlexWaygood** [comment]
I'm closing this PR, since it has been 2 years and @pitrou's review comments have still not been addressed.

@agfor: if you're still interested in working on this issue, please feel free to continue discussing it at #77621 and/or submit a new PR 🙂

**Pull request closed without merging**
